src/Hedgehog/Outcome.fs

```diff
@@ -7,14 +7,19 @@ type Outcome<'a> =
 
 module Outcome =
 
-    let cata result failure discard success =
-        match result with
+    let private cata
+        (outcome : Outcome<'a>)
+        (failure : unit -> 'b)
+        (discard : unit -> 'b)
+        (success :   'a -> 'b)
+            : 'b =
```

Comment:
I don't think we need to whitespace align `'a -> 'b`, and as a personal preference, I'd put `: 'b =` on the previous line.

See `takeSmallest` in `Property.fs`.

Reply:
https://github.com/hedgehogqa/fsharp-hedgehog/pull/285/commits/68d4dc2bec0b0d38ab76007200ec3f127c227bdf